decidim-demographics/lib/decidim/demographics/engine.rb

```diff
@@ -33,7 +33,7 @@ class Engine < ::Rails::Engine
         end
       end
 
-      initializer "decidim_demographics.webpacker.assets_path" do
+      initializer "decidim_demographics.shakapacker.assets_path" do
```

Comment:
Just a quick question on how to start handling this deprecations: should we still leave this initializer, show a deprecation warning in the logs and call to the new initializer as we do with the `Runner` module? (This would apply to all the Decidim engines/components of course, but I'm just leaving this comment one time)

Reply:
The only reason why i would use this kind of deprecation is to make sure that other plugin developers do not have their gems hooked to ours. Otherwise i do not see a real reason why we should ship the app with deprecation warnings that may not be needed.